iOS_SDK/OneSignalSDK/Source/OneSignal.h

```diff
@@ -141,6 +145,22 @@ typedef void (^OSWebOpenURLResultBlock)(BOOL shouldOpen);
 typedef void (^OSResultSuccessBlock)(NSDictionary* result);
 typedef void (^OSFailureBlock)(NSError* error);
 
+@protocol OSInAppMessages <NSObject>
+
++ (void)addTrigger:(NSString * _Nonnull)key withValue:(id _Nonnull)value;
++ (void)addTriggers:(NSDictionary<NSString *, id> * _Nonnull)triggers;
++ (void)removeTriggerForKey:(NSString * _Nonnull)key;
++ (void)removeTriggersForKeys:(NSArray<NSString *> * _Nonnull)keys;
++ (void)clearTriggers;
+// Allows Swift users to: OneSignal.InAppMessages.Paused = true
++ (BOOL)paused NS_REFINED_FOR_SWIFT;
++ (void)paused:(BOOL)pause NS_REFINED_FOR_SWIFT;
+
+typedef void (^OSInAppMessageClickBlock)(OSInAppMessageAction * _Nonnull action);
++ (void)setInAppMessageClickHandler:(OSInAppMessageClickBlock _Nullable)block;
```

Comment:
Now that this is under the `InAppMessage` namespace this should just be `setClickHandler`.